lib/game.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+require 'frame'
+
+class Game
+  attr_reader :frame, :scores
+
+  TP = 10
+  TF = 10
```

Comment:
Why are these two constants separate but of the same value?

Reply:
It would be easier if the constants had explicit names - using abbreviations are not recommended as it make it difficult to know what variables contain.

See this article for more on naming: https://medium.com/swlh/an-opinionated-guide-to-naming-your-code-aimed-at-new-developers-6f769d60e098